main.go

```diff
@@ -107,6 +108,7 @@ func main() {
 
 	pflag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
 	pflag.StringVar(&probeAddr, "health-probe-addr", ":8081", "The address the probe endpoint binds to.")
+	pflag.StringVar(&pprofAddr, "pprof-addr", "", "The address to expose the pprof server.")
```

Comment:
should we set a default here?

Reply:
The default is an empty string. When the empty string is provided to the manager, the server is not started (the default).

Reply:
can we add that to the docstring here? something like "default is empty string which disables the pprof server"

Reply:
Sure!